lib/ex.coffee

```diff
@@ -32,21 +32,19 @@ trySave = (func) ->
 
   deferred.promise
 
-saveAs = (filePath) ->
-  editor = atom.workspace.getActiveTextEditor()
-  fs.writeFileSync(filePath, editor.getText())
```

Comment:
I'm not sure about other purposes for it, but I know that when I used that saveAs rather than editor.saveAs Atom acted like the file wasn't saved (there was an asterisk next to the filename and you couldn't close the window without being prompted to save).